tfjs-core/src/serialization.ts

```diff
@@ -54,6 +54,13 @@ export declare type SerializableConstructor<T extends Serializable> = {
 export declare type FromConfigMethod<T extends Serializable> =
     (cls: SerializableConstructor<T>, config: ConfigDict) => T;
 
+const GLOBAL_CUSTOM_OBJECT =
```

Comment:
add comments for explain the usages of these two global map

Reply:
SG, added the explanation and example usage